# GetCameraInfo.srv
# Retrieve the current parameters for the camera
---
sensor_msgs/CameraInfo info
